src/app/health/[id]/page.tsx

```diff
@@ -1,39 +1,49 @@
 "use client";
 
```

Comment:
Unrelated to this file, but let's continue to follow proper branch naming convention

feature/
bugfix/
test/
refactor/
etc. branch names don't have to be long, just specific enough for documentation